src/components/CreateTask.tsx

```diff
@@ -1,6 +1,10 @@
 import React, { useState } from 'react'
 
-export default function CreateTask({ addTask }) {
+type Props = {
+  addTask: Function
+}
+
+export const CreateTask = ({ addTask }: Props) => {
```

Comment:
Don't forget to update the Props interface name here. And depending on how strict you want your typechecking to be, you'll want to add explicit return types. In the case of a React element, `JSX.Element` does the job.

For more (and other good suggestions), see the [React/Typescript cheatsheet](https://github.com/typescript-cheatsheets/react#function-components)


```suggestion
export const CreateTask = ({ addTask }: CreateTaskProps): JSX.Element => {
```